### ISSUE TITLE
Rails engine? 

### ISSUE THREAD

**Issue by jamesvanmil** [ActiveTriples/linked-data-fragments#8]
Following the lead of Questioning Authority, I think this should be a Rails engine. Any good reason not to do this?

**Comment by tpendragon** [comment]
With the idea being that you can mount the app in other apps? I'm not particularly against it, just no experience in writing them. Can you spin it up as an engine?

**Comment by jamesvanmil** [comment]
Yeah that's my thought. I think I can modify the existing code base to enginfy it without interfering with other work.

**Comment by no-reply** [comment]
I'm working on a refactor that makes these fragments mountable. This may take the form of a larger refactor that moves the core code out into a Rack module. Updates to come.

**Comment by no-reply** [comment]
There's a branch that makes this mountable at: https://github.com/ActiveTriples/linked-data-fragments/compare/feature/rack-server

Mounting instruction are at: https://github.com/ActiveTriples/linked-data-fragments/compare/feature/rack-server#diff-a334eb2b47b576b3194fbb8e04a94474R11

**Comment by hackartisan** [comment]
@no-reply is this issue closed? LDCX notes reference 'axing rails engine' which would imply yes.

However, notes from LDCX indicate that there is a bit of final alignment to do; can you give more details on whether that is a matter of PRing existing work or creating a new ticket to... discuss? take some (what?) action?

"Other topic with respect to LDF gem, Tom took logic out of rails part of that gem and in DCE branch, there are 2 equivalent server applications. Share all logic except routing. Would advocate possibly axing rails engine, no reason you can’t mount any rails app you like to rack. Want buy-in for rack instead of rails (rails is heavyweight for this work context)" (from https://docs.google.com/document/d/1yr7dIlKzWmaVLf-2easbaQvY5UF8tf0_ylr8RIKWprY/edit)

**Comment by jcoyne** [comment]
Hmm, I was unable to be at the LDCX discussion, but I'm still leaning towards possibly making this an engine as it would make the deployment story much easier.

**Comment by jcoyne** [comment]
I would like to hear more about the non-rails server applications that @no-reply has devised.
